mdx/api/index.mdx

```diff
@@ -0,0 +1,886 @@
+| `makerAddress`          | The party that creates the order. The maker is also one of the two parties that will be involved in the trade if the order gets filled.                                                                                                                                                                                                |
+| `takerAddress`          | The party that is allowed to fill the order. If set to a specific party, the order cannot be filled by anyone else. If left unspecified, anyone can fill the order.                                                                                                                                                                    |
+| `makerAssetData`        | Contains all the identifying information about the asset(s) the order maker is trying to sell.                                                                                                                                                                                                                                         |
+| `takerAssetData`        | Identifying information of the asset(s) the taker must trade in exchange for the maker's asset(s).                                                                                                                                                                                                                                     |
+| `makerAssetAmount`      | Amount of the maker's asset(s) being offered by the maker.                                                                                                                                                                                                                                                                             |
+| `takerAssetAmount`      | Amount of the taker's asset(s) the maker will accept in exchange for their maker asset(s). In order to calculate the price the maker is offering, one can divide the `makerAssetAmount` by the `takerAssetAmount` (the calculation is [a bit more complex](/docs/guides/v3-specification#assetproxy) if multiple assets are involved). |
+| `expirationTimeSeconds` | Timestamp in seconds of when the order expires. Expired orders cannot be filled.                                                                                                                                                                                                                                                       |
+| `salt`                  | A value that can be used to guarentee order uniqueness. Typically it is set to a random number.                                                                                                                                                                                                                                        |
```

Comment:
```suggestion
| `salt`                  | A value that can be used to guarantee order uniqueness. Typically it is set to a random number.                                                                                                                                                                                                                                        |
```

Do we have this in other places as well

Reply:
In core-concepts. Lots of fixes going into core-concepts...